apollo-router/src/state_machine.rs

```diff
@@ -324,6 +367,7 @@ where
 
         // Process all the events in turn until we get to error state or we run out of events.
         while let Some(event) = messages.next().await {
+            tracing::debug!("got event {}", event);
```

Comment:
```suggestion
```

Reply:
There's a specific ask that we can see the state machine update during manual testing. I can improve the message though.

Reply:
[e44556e](https://github.com/apollographql/router/pull/2596/commits/e44556e856aaffaf8ca4f1b6e7ec03cba9045c80)